### DISCUSSION TITLE
google-usmierca-kolejna-usluge-domeny-google-domains/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#108] [General]
# google-usmierca-kolejna-usluge-domeny-google-domains/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/google-usmierca-kolejna-usluge-domeny-google-domains/

**Comment by spexpl** [comment]
Ja domeny kupuję w OVH, a parkuję je na Cloudflare i mi taki team jak najbardziej pasuje.

**Comment by spexpl** [comment]
Namesco to taki angielski home i nazwa?

**Reply by idarek** [reply]
Tak.
